modules/ts/tse/functions.cpp

```diff
@@ -0,0 +1,519 @@
+	if (!Player(pid)->IsAlive()) return -1;
+	return Player(pid)->GetFreeSlots();
+}
+
+static cell AMX_NATIVE_CALL tse_setuserslots(AMX *amx, cell *params) 
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return 0;
+	if (!Player(pid)->IsAlive()) return 0;
+	Player(pid)->SetFreeSlots(params[2]);
+	return 1;
+}
+
+static cell AMX_NATIVE_CALL tse_getusercash(AMX *amx, cell *params) 
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return -1;
+	if (!Player(pid)->IsAlive()) return -1;
+	return Player(pid)->GetCash();
+}
+
+static cell AMX_NATIVE_CALL tse_setusercash(AMX *amx, cell *params) 
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return 0;
+	if (!Player(pid)->IsAlive()) return 0;
+	Player(pid)->SetCash(params[2]);
+	return 1;
+}
+
+static cell AMX_NATIVE_CALL tse_getuserkevlar(AMX *amx, cell *params)
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return -1;
+	if (!Player(pid)->IsAlive()) return -1;
+	return (bool)(Player(pid)->GetPDataInt(628) != 0);
+}
+
+static cell AMX_NATIVE_CALL tse_getuserstamina(AMX *amx, cell *params) 
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return -1;
+	if (!Player(pid)->IsAlive()) return -1;
+	return Player(pid)->GetMeleeStamina();
+}
+
+static cell AMX_NATIVE_CALL tse_setuserstamina(AMX *amx, cell *params) 
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return 0;
+	if (!Player(pid)->IsAlive()) return 0;
+	Player(pid)->SetMeleeStamina(amx_ctof(params[2]));
+	MESSAGE_BEGIN(MSG_ONE, GetMsgIDByName("KFuPower"), 0, Player(pid)->PlayerEdict);
+		WRITE_BYTE((byte)round(amx_ctof(params[2])));
+	MESSAGE_END();
+	return 1;
+}
+
+static cell AMX_NATIVE_CALL tse_geteffectsphysspd(AMX *amx, cell *params)
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return -1;
+	if (!Player(pid)->IsAlive()) return -1;
+	return Player(pid)->GetEffectsPhysicsSpeed();
+}
+
+static cell AMX_NATIVE_CALL tse_seteffectsphysspd(AMX *amx, cell *params)
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return 0;
+	if (!Player(pid)->IsAlive()) return 0;
+	Player(pid)->SetEffectsPhysicsSpeed(amx_ctof(params[2]));
+	return 1;
+}
+
+static cell AMX_NATIVE_CALL tse_getuserphysspd(AMX *amx, cell *params)
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return -1;
+	if (!Player(pid)->IsAlive()) return -1;
+	return Player(pid)->GetPlayerPhysicsSpeed();
+}
+
+static cell AMX_NATIVE_CALL tse_setuserphysspd(AMX *amx, cell *params)
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return 0;
+	if (!Player(pid)->IsAlive()) return 0;
+	Player(pid)->SetPlayerPhysicsSpeed(amx_ctof(params[2]));
+	return 1;
+}
+
+static cell AMX_NATIVE_CALL tse_getusercurritems(AMX *amx, cell *params)
+{
+	byte pid = params[1];
+	if (!IsPlayerValid(amx, pid)) return -1;
+	if (!Player(pid)->IsAlive()) return -1;
+	switch ((uint32_t)Player(pid)->GetPDataInt(462))
+	{
+		case 65536:
+			return 1;
+			break;
+		case 16777216:
+			return 2;
+			break;
+		case 16842752:
```

Comment:
I think it's better to replace raw numbers with meaningful defines and `break` after `return` is redundant.